motor/aiohttp/__init__.py

```diff
@@ -190,7 +191,18 @@ async def __call__(self, request):
             raise aiohttp.web.HTTPNotFound(text=request.path)
 
         resp = aiohttp.web.StreamResponse()
-        self._set_standard_headers(request.path, resp, gridout)
+
+         # Calculate the md5 for the GridFS file.
+        md5 = hashlib.md5()
```

Comment:
We shouldn't generate md5 sums. See https://jira.mongodb.org/browse/MOTOR-192, and there should be (but it doesn't look like there is) a motor equivalent to https://jira.mongodb.org/browse/PYTHON-2998.

Reply:
I've removed the `disable_md5` option and `md5` properties everywhere, in this case it is used for the [`Etag`](https://httpwg.org/specs/rfc7232.html#header.etag) `http` header, which is for the web server's use primarily.   The [spec](https://httpwg.org/specs/rfc7232.html#header.etag) doesn't dictate what algorithm to use, but `tornado` uses `sha1` and `CRC32C` is another good candidate (recommended by [Google Cloud](https://cloud.google.com/storage/docs/hashes-etags)).

Reply:
Actually, if the motivation is strong FIPS [compliance](https://www.ibm.com/docs/sr/itcam-app-mgr/7.2.1?topic=aic-complying-fips-requirements), we should use `sha256`, I'll update to that.

Reply:
Nice catch! I responded to this thread here: https://github.com/mongodb/motor/pull/141#discussion_r774145847